src/components/Legend/LegendFlexBoxes.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components';
+import { IsActiveProps } from './MapLayersLegend';
+export interface FlexColumnProps {
+  isLast?: boolean;
+}
+export const FlexColumnLast = styled.div<FlexColumnProps>`
+  display: flex;
+  flex-direction: column;
+  ${props => (props.isLast === true ? 'margin-top:auto;' : '')}
+`;
+export const FlexRow = styled.div`
```

Comment:
Mah...Still not convinced about their naming here but maybe that's something for another time.

Reply:
Me neither, but as I said. I'm actually not sure what would be the right naming. These are just some layout classes nearly no logic here.